### PR TITLE
Added Ability to move  topic to a different forum. 

### PR DESCRIPTION
**Pull request by gabrielbaez**

### CODE REVIEW

File: app/views/forem/admin/topics/_form.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <%= simple_form_for [:admin, @topic] do |f| %>
   <%= f.input :subject %>
+  <%= f.association :forum , :label => "Forum" %>
```
Comment (gabrielbaez) [inline app/views/forem/admin/topics/_form.html.erb:3]: I had to specify the label here  because without it, it was outputting "Title,  description"  text.

**Comment by radar** [comment]
Could you add a test for this please?

**Comment by gabrielbaez** [comment]
I can, but it is a bit difficult since rake routes  does not  work  to get the list of  paths  
am I running this wrong ?  
gabriel@railsdev:~/forem$ rake routes
rake aborted!
Don't know how to build task 'routes'

(See full trace by running task with --trace)

**Comment by reset** [comment]
@gabrielbaez when working with an engine that uses a dummy for testing you have to access the Rake tasks under the app namespace

`rake app:routes`

Still, you might not get the output you are expecting since Forem is a fully isolated engine.

**Comment by radar** [comment]
That will give you the dummy application's routes, but not the engines. I've just added `rake forem:routes` in @dabb3b62d7b4f648d8799499992b605f7e5cf6d5 that will give you forem's routes.

**Comment by reset** [comment]
Weird that theres no built in way to get an output of an Engines routes in Rails yet

**Comment by gabrielbaez** [comment]
Thanks for the help,  added the test let me know how it looks it passed on my end.

**Comment by gabrielbaez** [comment]
@radar  added the check,  thanks.

**Comment by radar** [comment]
Great, thanks for your work!

**Pull request merged**